recipes/protobuf-python/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conan import ConanFile
+from conan.tools.layout import basic_layout
+from conan.tools.files import copy, get
+
+import os
+import sys
+
+required_conan_version = ">=1.53"
+
+class ProtobufPythonPkg(ConanFile):
+    name = "protobuf-python"
+    description = "Protocol Buffers - Google's data interchange format - Python Module"
+    topics = ("protocol-buffers", "protocol-compiler", "serialization", "rpc", "protocol-compiler", "python")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/protocolbuffers/protobuf"
+    license = "BSD-3-Clause"
+    no_copy_source = True
+    settings = "os", "arch", "compiler", "build_type"
+
+    def layout(self):
+        basic_layout(self,src_folder="src")
+
+    def requirements(self):
+        self.requires(f"protobuf/{self.version}", run=True, build=True)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def build(self):
+        self.run(f"{sys.executable} setup.py build",
```

Comment:
Is there any reason to only execute "setup.py build" and not "setup.py install" too?

Also, why not make this part of protobuf recipe as a "with_python" option?